app/views/radius/_topnavbar.html.erb

```diff
@@ -0,0 +1,111 @@
+        <a class="hidden-xs" data-toggle-state="aside-collapsed" href="#">
+          <em class="fa fa-navicon">
+          </em>
+        </a>
+        <!--Button to show/hide the sidebar on mobile. Visible on mobile only.-->
+        <a class="visible-xs sidebar-toggle" data-no-persist="true" data-toggle-state="aside-toggled" href="#">
+          <em class="fa fa-navicon">
+          </em>
+        </a>
+      </li>
+      <li>
+        <a href="//store.radiusnetworks.com" title="Radius Networks Store">
+          <em class="icon-basket">
+          </em>
+        </a>
+      </li>
+    </ul>
+    <ul class="nav navbar-nav navbar-right">
+      <!--Search icon-->
+      <!--li-->
+      <!-- a data-search-open="" href="#"-->
+      <!--   em.icon-magnifier-->
+      <!--User Drop Down-->
+      <li class="dropdown dropdown-list">
+        <a data-toggle="dropdown" href="#">
+          <em class="icon-user">
+          </em>
+        </a>
+        <!--START Dropdown menu-->
+        <ul class="dropdown-menu animated flipInX">
+          <li>
+            <!--START list group-->
+            <div class="list-group">
+              <a class="list-group-item" href="<%= signout_user_path %>">
+                <div class="media-box">
+                  <div class="pull-left">
+                    <em class="fa fa-sign-out fa-2x text-info">
+                    </em>
+                  </div>
+                  <div class="media-box-body clearfix">
+                    <p class="m0">Sign Out</p>
+                    <p class="m0 text-muted">
+                    <small>Signed in as <%= current_user.email %></small>
+                    </p>
+                  </div>
+                </div>
+              </a>
+              <!--list item-->
+              <a class="list-group-item" href="<%= manage_user_path %>">
+                <div class="media-box">
+                  <div class="pull-left">
+                    <em class="fa fa-user fa-2x text-success">
+                    </em>
+                  </div>
+                  <div class="media-box-body clearfix">
+                    <p class="m0">User Account</p>
+                    <p class="m0 text-muted">
+                    <small>Edit Account Details</small>
+                    </p>
+                  </div>
+                </div>
+              </a>
+              <!--list item-->
+              <a class="list-group-item" href="<%= manage_teams_path %>">
+                <div class="media-box">
+                  <div class="pull-left">
+                    <em class="fa fa-users fa-2x text-warning">
+                    </em>
+                  </div>
+                  <div class="media-box-body clearfix">
+                    <p class="m0">Teams</p>
+                    <p class="m0 text-muted">
+                    <small>
+                      You belong to <%= pluralize(current_user.accounts.count ,"team") %>
```

Comment:
This will have problems on new apps as they do not have `Account` any more. Just teams.